### PR TITLE
Support for iOS Fullscreen

### PR DESCRIPTION
**Pull request by dhalloop7**
Removed condition for android only.
Working for iOS devices as well.

### CODE REVIEW

File: lib/src/chewie_player.dart [hunk 1]
```diff
@@ -127,12 +127,12 @@ class ChewieState extends State<Chewie> {
     );
 
     SystemChrome.setEnabledSystemUIOverlays([]);
-    if (isAndroid) {
+//    if (isAndroid) {
```
Comment (AndroidNils) [inline lib/src/chewie_player.dart:130]: Why do you comment it out instead of removing this lines?

File: lib/src/chewie_player.dart [hunk 2]
```diff
@@ -127,12 +127,12 @@ class ChewieState extends State<Chewie> {
     );
 
     SystemChrome.setEnabledSystemUIOverlays([]);
-    if (isAndroid) {
```
Comment (AndroidNils) [inline lib/src/chewie_player.dart:130]: What was the reason for only supporting this feature on Android?

**Comment by w3ggy** [comment]
Could you fix this MR and merge it into the repo?

**Comment by roskimlong** [comment]
Work well on the video link, but not work on live link :(

**Comment by Ahmadre** [comment]
Hey guys 😊 This issue is already fixed with a big PR a day ago. Enjoy cross-platform 👌🏽

**Pull request closed without merging**